portfolio/src/main/webapp/index.html

```diff
@@ -2,14 +2,32 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Nitin's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+
+  <div id="mySidenav" class="sidenav">
+  <a href="javascript:void(0)"</a>
+  <a href="#">About</a>
+  <a href="#">Contact</a>
+  <a href="#">Goals</a>
+  </div>
+
+  <body>  
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Nitin's Portfolio</h1>
+      <marquee>Latest Update: Learning web app development and practicing machine learning</marquee>
+      <hr style="height:1px;border-top:1px solid #0000FF" />
+      <img src="images/profile.jpg" alt="Profile Photograph">
+      <h3>University of Florida Bachelors in Computer Science</h3>
+      <h3>Google STEP Intern</h3>
+      <p>I have been learning how to program for roughly 6 years now. I began with learning web development for an FBLA competition and I now build android applications for national competitions like PennApps and for my personal developer account on the Play Store. Software development is one of my favorite subfields in computer science. Throughout my junior and senior year in high school, I taught students how to program in a computer science club that I founder. My goal is to lead emerging technologies and build software that can make a postive impact in society.</p>
```

Comment:
For long paragraphs like this, you should multiline them.
\<p>
[indent]I have been learning...
\</p>

Your text editor might have maximum spacing settings, generally stops at 80 characters. Going beyond that can get hard to read.

Reply:
I will do that in the next pull request, thank you!